internal/provider/provider.go

```diff
@@ -117,14 +122,53 @@ func (p *pingfederateProvider) Schema(_ context.Context, _ provider.SchemaReques
 				MarkdownDescription: "Path for PingFederate Admin API. Default value can be set with the `PINGFEDERATE_PROVIDER_ADMIN_API_PATH` environment variable. If no value is supplied, the value used will be `/pf-admin-api/v1`.",
 				Optional:            true,
 			},
+			"access_token": schema.StringAttribute{
+				MarkdownDescription: "Access token for PingFederate Admin API. Default value can be set with the `PINGFEDERATE_PROVIDER_ACCESS_TOKEN` environment variable.",
+				Optional:            true,
+				Sensitive:           true,
+				Validators: []validator.String{
+					stringvalidator.ConflictsWith(path.MatchRoot("oauth")),
+					stringvalidator.ConflictsWith(path.MatchRoot("username")),
+					stringvalidator.ConflictsWith(path.MatchRoot("password")),
+				},
+			},
 			"username": schema.StringAttribute{
 				MarkdownDescription: "Username for PingFederate Admin user. Default value can be set with the `PINGFEDERATE_PROVIDER_USERNAME` environment variable.",
 				Optional:            true,
+				Validators: []validator.String{
+					stringvalidator.ConflictsWith(path.MatchRoot("oauth")),
```

Comment:
For these conflicts, it would be good to describe the conflict cases in the attribute description, since the errors terraform outputs on these tend to be a little cryptic